lib/white_bread/context/step_executor.ex

```diff
@@ -1,6 +1,23 @@
 defmodule WhiteBread.Context.StepExecutor do
   alias WhiteBread.RegexExtension
 
+  defprotocol ErrorHandler do
+    @fallback_to_any true
+    def get_tuple(error, step, _stacktrace)
```

Comment:
I think this should be ```stacktrace``` not ```_stacktrace``` as each implementation can choose to use it or not.